Library not working when included with AWS Layers

Reproduction Steps

1> Create a Lambda Function(https://docs.aws.amazon.com/lambda/latest/dg/getting-started-create-function.html)
2> Create a layer and associate it with your lambda created in Step 1(https://docs.aws.amazon.com/lambda/latest/dg/configuration-layers.html). The purpose of layers is to have a shared library across multiple AWS Lambda functions. So here I am putting the future-fstrings library as a part of my zip file which is uploaded as AWS layer.

When I run the function I get the following error
`[ERROR] Runtime.UserCodeSyntaxError: Syntax error in module 'main': unknown encoding for '/var/task/main.py': future_fstrings
Traceback (most recent call last):
  File "None" Line None
    None`

Assuming main.py contains my lambda function code
If I include the future_strings in same folder as my code then it works. 
AWS Lambda puts the shared libraries in /opt/ folder and I am dynamically adding opt directory to sys path `sys.path.insert(0, '/opt') ` Doing this way it throws error. 

 I do not know the internals how the library converts and when does the conversion code triggers but my hunch is I am doing something wrong here.

--- a/README.md
+++ b/README.md
@@ -78,6 +78,25 @@ future-fstrings-show code.py > code_rewritten.py
       f-strings.
 2. A `.pth` file which registers a codec on interpreter startup.
 
+## when you aren't using normal `site` registration
+
+in setups (such as aws lambda) where you utilize `PYTHONPATH` or `sys.path`
+instead of truly installed packages, the `.pth` magic above will not take.
+
+for those circumstances, you'll need to manually initialize `future-fstrings`
+in a non-fstring wrapper.  for instance:
+
+```python
+import future_fstrings
+
+future_fstrings.register()
+
+from actual_main import main
+
+if __name__ == '__main__':
+    exit(main())
+```
+
 ## you may also like
 
 - [future-breakpoint](https://github.com/asottile/future-breakpoint)

